components/bl-list-component/styles/index.less

```diff
@@ -0,0 +1,24 @@
+@bl-customComponent-list-theme: @themePrimary;
+@bl-customComponent-list-themeTextColor: @appTextColor;
+
+@bl-customComponent-list-margin: 5px;
+@bl-customComponent-list-item-fontSize: 1rem;
+@bl-customComponent-list-item-line-height: 1.3;
+@bl-customComponent-list-item-padding: 3px;
+
+@bl-customComponent-list-item-cursor: pointer;
+
+.bl-customComponent-list {
+  color: @bl-customComponent-list-themeTextColor;
+
+  .list {
+    margin: @bl-customComponent-list-margin;
+
+    &__item {
```

Comment:
I would recommend you avoid deep nesting, in the CSS it will look like:

```
.bl-customComponent-list .list {...}
.bl-customComponent-list .list .list__item {...}
```

what about:

```
.bl-customComponent-list {

  .list{} 

  .list__item{}
}
```